### PR TITLE
feat: added a download link for global statistics

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
Adds a download link for the global statistics csv

https://unep-wcmc.codebasehq.com/projects/protected-planet-support-and-maintenance/tickets/261

Testing:

go to the home page, click the download link

image

### CODE REVIEW

File: app/controllers/global_statistics_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class GlobalStatisticsController < ApplicationController
+  def show
```
Comment (lucacug) [inline app/controllers/global_statistics_controller.rb:2]: can we rename this action to something like `download` ?

File: config/routes.rb [hunk 2]
```diff
@@ -34,6 +34,8 @@
     get '/country/:iso/compare(/:iso_to_compare)', to: 'country#compare', as: 'compare_countries'
     get '/country/:iso/protected_areas', to: 'country#protected_areas', as: 'country_protected_areas'
 
+    get '/global_statistics', to: 'global_statistics#show'
```
Comment (lucacug) [inline config/routes.rb:37]: change this accordingly to the controller action name comment

File: lib/modules/wdpa/global_stats_importer.rb [hunk 3]
```diff
@@ -10,7 +10,10 @@ def self.import
     CSV.foreach(latest_global_statistics_csv, headers: true) do |row|
       field = row['type']
       value = parse_value(row['value'])
-      attrs.merge!("#{field}": value)
+
+      # The global_statistics csv can now be downloaded so a methodology url has been added
+      # to the end of the spreadsheet. We need to not add this line to the attributes.
+      attrs.merge!("#{field}": value) if field.present?
```
Comment (lucacug) [inline lib/modules/wdpa/global_stats_importer.rb:16]: yeah it makes sense

**Review by lucacug — CHANGES_REQUESTED** [review]
just a small name change, after that it is good to go

File: app/controllers/global_statistics_controller.rb (hunk 1 above)
Comment (sergiomarrocoli) [reply app/controllers/global_statistics_controller.rb]: changed

File: config/routes.rb (hunk 2 above)
Comment (sergiomarrocoli) [reply config/routes.rb]: changed

**Review by lucacug — APPROVED** [review]

**Pull request merged**